src/routes/papers/index.js

```diff
@@ -71,10 +93,24 @@ function getUserBadgesByType(request, response) {
     return;
   }
 
-  returnBadges(badgerService.getBadges(request.params.orcid, request.params.badge, {
+  return badgerService.getBadges(request.params.orcid, request.params.badge, {
     '_1': request.params.doi1,
     '_2': request.params.doi2
-  }), request, response);
+  });
+}
+
+function getUserBadgesByBadge(request, response) {
+  returnBadges(getUserBadgesByType(request, response), request, response);
+}
+
+function getUserBadgesByBadgeCount(request, response) {
```

Comment:
I see. Good, so I'll change the assert to `<= 1` for this one. won't move things around till you finish the review though, so we don't get lost in changes.